pytorch_blade/torch_blade/dynamo/__init__.py

```diff
@@ -59,131 +63,140 @@ def disc_compile(fx_g: fx.GraphModule, inps) -> Callable:
             node.kwargs = new_kwargs
 
         fx_g.graph.lint()
-
         fx_g.recompile()
         f = torch.jit.script(fx_g)
-
         torch._C._jit_pass_remove_mutation(f.graph)
 
         f = torch.jit.freeze(f.eval())
-        f = torch.jit.optimize_for_inference(f)
+        if use_ts:
+            return f
         cfg = torch_blade.Config()
-        cfg.disable_optimization_for_inference = True
+        cfg.disable_optimization_for_inference = is_training
         with cfg:
             f = torch_blade.optimize(f, True, tuple(inps))
+        with open('aot_disc.py', 'a') as writer:
+            writer.write(str(f.forward.code) + '\n')
+
     return f
 
+@compilers.make_boxed_compiler
+def disc_compile(fx_g: fx.GraphModule, inps, use_ts=False) -> Callable:
+    return _disc_compile(fx_g, inps, use_ts=False)
```

Comment:
`use_ts` argument on L83 is never used ...

Reply:
It's used. See L91